### PR TITLE
Tamas

### PR DESCRIPTION
**Pull request by Egrishenka**

### CODE REVIEW

File: src/main/java/com/egrine/mailspammer/emailrecipient/EmailRecipientRepository.java [hunk 1]
```diff
@@ -8,9 +8,25 @@
 @Repository
 public interface EmailRecipientRepository extends JpaRepository<EmailRecipient, Long> {
 
-    @Query(value="SELECT email_recipient.id, email_recipient.email_address, email_recipient.is_active" +
+    @Query(value="SELECT email_recipient.id, email_recipient.email_address, email_recipient.is_active, " +
```
Comment (WDev-Beginner) [inline src/main/java/com/egrine/mailspammer/emailrecipient/EmailRecipientRepository.java:11]: extra comma

File: src/main/java/com/egrine/mailspammer/emailrecipient/implementation/InMemoryEmailRecipientService.java [hunk 2]
```diff
@@ -1,23 +1,100 @@
...
     EmailRecipientRepository repository;
+    UserService userService;
+    EmailService emailService;
 
     @Override
     public List<EmailRecipient> getEmailRecipientsById(List<Long> recipientIds) {
         List<EmailRecipient> recipients = repository.getEmailRecipientsById(recipientIds);
         return recipients == null ? Collections.emptyList() : recipients;
     }
+
```
Comment (WDev-Beginner) [inline src/main/java/com/egrine/mailspammer/emailrecipient/implementation/InMemoryEmailRecipientService.java:35]: service layer should throw exception if the recipient is not found, you are not using the exception you defined

**Review by WDev-Beginner — CHANGES_REQUESTED** [review]
Checkout the comments on the code!

File: src/main/java/com/egrine/mailspammer/emailrecipient/implementation/InMemoryEmailRecipientService.java (hunk 2 above)
Comment (Egrishenka) [reply src/main/java/com/egrine/mailspammer/emailrecipient/implementation/InMemoryEmailRecipientService.java:35]: the exception is used in the update function, just like in the templates

**Pull request merged**